src/main/java/duke/Deadline.java

```diff
@@ -0,0 +1,34 @@
+package duke;
+
+import java.time.LocalDate;
+
+/**
```

Comment:
I like how most public method and classes have JavaDocs comments making it easier to read your code. 👍